lib/response-builder.ts

```diff
@@ -28,18 +28,41 @@ function buildResponse(
   const applicationScans: types.ScannedProjectCustom[] =
     depsAnalysis.applicationScans || [];
 
+  const scannedProjects = [
+    {
+      packageManager: plugin.packageManager,
+      depTree: pkg,
+    },
+    ...applicationScans,
+  ];
+
+  const groupedScannedProjects = groupScannedProjects(
```

Comment:
I think groupScannedProjects is confusing, because we dont group them here, we just adding meta part, so maybe something like addMetaToScannedProjects?